### PR TITLE
Create a true Makefile with Version management

### PR DESCRIPTION
**Pull request by mathieu-pousse**
Signed-off-by: Mathieu POUSSE <mathieu.pousse@zenika.com>

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -12,6 +13,8 @@ var RootCmd = &cobra.Command{
 	Long: `A better dart game than the chinese one.
 	Complete doc at voir http://github.com/Zenika/go-dart.`,
```
Comment (ggerbaud) [inline cmd/root.go]: C'est l'occasion de mettre a jour cette url

Comment (mathieu-pousse) [inline cmd/root.go]: definitly, and to use http*s* !

**Comment by mathieu-pousse** [comment]
$ ./dist/go-dart version
Version:     v0.1.0-alpha
Git hash:    d440579d7fcfe950a3706a57dacacab1cb6dc3dd
Build Date:  2016-07-12T08:34:19Z

Project Url: https://github.com/gocaine/go-dart

**Comment by ggerbaud** [comment]
LGTM

**Pull request merged**